notebook_index.yml

```diff
@@ -115,7 +115,7 @@ contents:
               - pump: Pump unit model with iapws property package
               - heat exchanger 0D: Heat Exchanger 0D unit model heating a benzene-toluene mixture using steam
               - skeleton unit: Skeleton unit model demonstrating custom variables, ports, and constraints
-              - statejunction: Statejunction unit model with modular property package
+              - statejunction: Statejunction with modular property package
```

Comment:
Typo: Capitalise J in StateJunction, This applies in a few place.

Reply:
I assume the first statejunction in line 118 is fine tho?